src/main/java/jenkins/metrics/api/MetricsRootAction.java

```diff
@@ -158,7 +158,7 @@ public HttpResponse doHealthcheck(StaplerRequest req, @QueryParameter("key") Str
             key = getKeyFromAuthorizationHeader(req);
         }
         Metrics.checkAccessKeyHealthCheck(key);
-        return Metrics.cors(key, new HealthCheckResponse(Metrics.healthCheckRegistry().runHealthChecks()));
+        return Metrics.cors(key, new HealthCheckResponse(Metrics.getHealthCheckResults()));
```

Comment:
The intent of these action methods was to actually block until the live result was available... I suspect this will now return the last cached value instead... which would be bad